pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -135,3 +135,12 @@ func (pl *NodeLabel) Score(ctx context.Context, state *framework.CycleState, pod
 func (pl *NodeLabel) ScoreExtensions() framework.ScoreExtensions {
 	return nil
 }
+
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *NodeLabel) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.Add},
```

Comment:
As this plugin's Filter implementation is only related to its plugin arguments - which labels to be present/absent. So a Pod event won't help in making this schedulable.